staging/src/k8s.io/component-base/metrics/histogram.go

```diff
@@ -21,17 +21,37 @@ import (
 
 	"github.com/blang/semver/v4"
 	"github.com/prometheus/client_golang/prometheus"
+	"go.opentelemetry.io/otel/trace"
 )
 
 // Histogram is our internal representation for our wrapping struct around prometheus
 // histograms. Summary implements both kubeCollector and ObserverMetric
 type Histogram struct {
+	ctx context.Context
 	ObserverMetric
 	*HistogramOpts
 	lazyMetric
 	selfCollector
 }
 
+func (h *Histogram) Observe(v float64) {
+	h.withExemplar(v)
+}
+
+func (h *Histogram) withExemplar(v float64) {
+	var exemplarLabels prometheus.Labels
```

Comment:
Should we move this inside the `if` statement below to avoid unnecessary computation?

Reply:
Sounds good, moved.